presto-accumulo/src/main/java/com/facebook/presto/accumulo/AccumuloConnectorFactory.java

```diff
@@ -56,7 +56,8 @@ public Connector create(String connectorId, Map<String, String> config, Connecto
             return injector.getInstance(AccumuloConnector.class);
         }
         catch (Exception e) {
-            throw Throwables.propagate(e);
+            throwIfUnchecked(e);
+            throw new RuntimeException(e);
```

Comment:
Yes, though we had this problem previously with `Throwables.propagate()`, and I'm not concerned about it here since failing to create a connector means the server will fail to start (and who would be interrupting the thread?)